pkg/operator/starter.go

```diff
@@ -7,6 +7,7 @@ import (
 
 	configv1 "github.com/openshift/api/config/v1"
 	operatorv1 "github.com/openshift/api/operator/v1"
+	"github.com/openshift/api/sharedresource"
```

Comment:
I assume we should do sharedresource/v1alpha1?
```suggestion
	"github.com/openshift/api/sharedresource/v1alpha1"
```

Reply:
nope ... https://github.com/openshift/cluster-storage-operator/pull/257#issuecomment-1020623049